include/yarp/util/yp_strnstr.h

```diff
@@ -0,0 +1,14 @@
+#ifndef YP_STRNSTR_H
+#define YP_STRNSTR_H
+
+#include "yarp/defines.h"
+
+#include <stddef.h>
+#include <string.h>
+
+// Here we have rolled our own version of strnstr because for the most part it
+// is not available unless on BSD, and even then -std=c99 sometimes does not
+// include it.
```

Comment:
It's available on Linux and macOS, so maybe the comment should just be clear it's really just indows missing it?

Reply:
Ah sorry, this is `strnstr`, not `snprintf`, nevermind